scripts/util/k8s.py

```diff
@@ -80,3 +80,35 @@ def get_latest_releases_by_minor() -> Dict[str, str]:
             latest_by_minor[key] = tag
 
     return latest_by_minor
+
+
+def get_all_releases_after(release) -> set[str]:
+    """
+    Get all releases after the input release.
+    If the input release is invalid, the output will be empty.
+    """
+    releases: set[str] = set()
+    version_regex = re.compile(r"^v?(\d+)\.(\d+)\..+")
+
+    match = re.match(r"^(\d+)\.(\d+)$", release)
+    if not match:
+        return releases
+    least_major, least_minor = match.groups()
+
+    for tag in get_k8s_tags():
```

Comment:
i'm concerned that `get_k8s_tags()` doesn't deal with pagination.  

```sh
$ curl -s https://api.github.com/repos/kubernetes/kubernetes/tags | jq '.[].name' | wc -l
30
```

that API we're using on github only returns the 30 tags -- not a full list of all k8s tags.  We likely will miss something things here because of that.